site/ts/activity-dashboard.js

```diff
@@ -130,10 +130,13 @@ export function applySettings() {
         const s_grad_sub = data[i].gradeable_submission;
         const s_forum_view = data[i].forum_view;
         const s_forum_post = data[i].forum_post;
-        const s_num_polls = data[i].num_poll_responses;
+        let s_num_polls = data[i].num_poll_responses;
         const s_off_hours = data[i].office_hours_queue;
-        let flag = false;
+        if (s_num_polls === null) {
+            s_num_polls = 0;
+        }
```

Comment:
You could use the [null(ish)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) coalescing operator for this.

Reply:
Unfortunately, as we figured out in the meeting yesterday, our linter doesn't recognize that operator. We need to update the linter.